code/modules/atmospherics/machinery/airalarm.dm

```diff
@@ -622,7 +622,8 @@
 		return
 
 	var/datum/gas_mixture/environment = location.return_air()
-	var/total = environment.oxygen + environment.nitrogen + environment.carbon_dioxide + environment.toxins
+	var/known_total = environment.oxygen + environment.nitrogen + environment.carbon_dioxide + environment.toxins
```

Comment:
might want to add N2O to this. it shows up as an "unknown" gas, not sure if this was intended or not. I doubt it considering its very well known.